keyboards/phage_studio/pila87/keymaps/horrortroll/keymap_stuff.h

```diff
@@ -0,0 +1,244 @@
+                dprintf("Gradient 100 HSV: %d, %d, %d\n", gradient_100.h, gradient_100.s, gradient_100.v);
+            }
+            return false;
+        case G2_VAD:
+            if (record->event.pressed) {
+                gradient_100.v = (gradient_100.v - color_adj_step * 2 >= 0) ? gradient_100.v - color_adj_step * 2 : 0;
+                dprintf("Gradient 100 HSV: %d, %d, %d\n", gradient_100.h, gradient_100.s, gradient_100.v);
+            }
+            return false;
+        case G_PRE:
+            if (record->event.pressed) {
+                gp_i = (gp_i + gp_length ) % gp_length;
+
+                gradient_0 = gradient_presets[gp_i].gradient_0;
+                gradient_100 = gradient_presets[gp_i].gradient_1;
+                reflected_gradient = gradient_presets[gp_i].reflected;
+
+                gp_i += 1;
+            }
+            return false;
+        case REF_G:
+            if (record->event.pressed) {
+                reflected_gradient = !reflected_gradient;
+            }
+            return false;
+        case G_FLIP:
+            if (record->event.pressed) {
+                HSV temp_color = gradient_0;
+                gradient_0 = gradient_100;
+                gradient_100 = temp_color;
+            }
+            return false;
+        case RGB_C_E:
+            if (record->event.pressed) {
+                switch (rgb_matrix_get_mode()) {
+                    case RGB_MATRIX_CUSTOM_CUSTOM_GRADIENT:
+                        rgb_matrix_mode(RGB_MATRIX_CUSTOM_COOL_DIAGONAL);
+                        return false;
+                    case RGB_MATRIX_CUSTOM_COOL_DIAGONAL:
+                        rgb_matrix_mode(RGB_MATRIX_CUSTOM_FLOWER_BLOOMING);
+                        return false;
+                    case RGB_MATRIX_CUSTOM_FLOWER_BLOOMING:
+                        rgb_matrix_mode(RGB_MATRIX_CUSTOM_KITT);
+                        return false;
+                    case RGB_MATRIX_CUSTOM_KITT:
+                        rgb_matrix_mode(RGB_MATRIX_CUSTOM_RANDOM_BREATH_RAINBOW);
+                        return false;
+                    default:
+                        rgb_matrix_mode(RGB_MATRIX_CUSTOM_CUSTOM_GRADIENT);
+                        return false;
+                }
+            }
+            return false;
+        case RGB_TOG:
+            if (record->event.pressed) {
+                switch (rgb_matrix_get_flags()) {
+                    case LED_FLAG_ALL: {
+                        rgb_matrix_set_flags(LED_FLAG_NONE);
+                        rgb_matrix_set_color_all(0, 0, 0);
+                    }
+                    break;
+                    default: {
+                        rgb_matrix_set_flags(LED_FLAG_ALL);
+                        rgb_matrix_enable_noeeprom();
+                    }
+                    break;
+                }
+            }
+            return false;
+	}
+    return true;
+}
+
+void rgb_matrix_indicators_user(void) {
```

Comment:
Note to self: #18450

Reply:
Done, i've just retarget to develop for new changes on that pr